source/_docs/scripts/service-calls.markdown

```diff
@@ -176,6 +176,86 @@ data:
 
 {% endraw %}
 
+### Use response data in templates: convert YAML to JSON
```

Comment:
goal is to explain in the HA documentation how to use the response data in ones templates, and how this can be done converting the presented Yaml to JSON and next, use that JSON data structure inside dv tools template